### ISSUE TITLE
Accuracy on testing_list.txt

### ISSUE THREAD

**Issue by LeighSu** [ARM-software/ML-KWS-for-MCU#61]
I have run the predefined model on the files named in testing_list.txt(under the speech_dataset), and the accuracy are  75.9% 79.2% 81.0% for DNN_S, DNN_M, DNN_L model, 86.3%  87.6% 90.4% for CNN_S, CNN_M, CNN_L model.

**Comment by navsuda** [comment]
@LeighSu,
I see the difference is because of (a) it was trained and tested on speech_commands_v0.01 (i.e. previous release of the dataset) and (b) the separation of dataset to train/val/test is based on a hash of the audio file names (see Custom Training Data section in [this tensorflow tutorial](https://www.tensorflow.org/tutorials/sequences/audio_recognition)), which may not exactly match with testing_list.txt.
